src/features/simd.md

```diff
@@ -18,28 +18,67 @@ The high-level goal of the WebAssembly SIMD proposal is to introduce vector oper
 
 The set of SIMD instructions is large, and varied across architectures. The set of operations included in the WebAssembly SIMD proposal consist of operations that are well supported on a wide variety of platforms, and are proven to be performant. To this end, the current proposal is limited to standardizing Fixed-Width 128-bit SIMD operations.
 
-The current proposal introduces a new v128 value type, and a number of new operations that operate on this type. The criteria used to determine these operations are:
+The current proposal introduces a new `v128` value type, and a number of new operations that operate on this type. The criteria used to determine these operations are:
 
 - The operations should be well supported across multiple modern architectures.
 - Performance wins should be positive across multiple relevant architectures within an instruction group.
 - The chosen set of operations should minimize performance cliffs if any.
 
-The proposal is in active development, both V8 and the toolchain have working prototype implementations for experimentation. As these are prototype implementations, they are subject to change as new operations are added to the proposal.
+The proposal is now in finalized state, both V8 and the toolchain have working implementations.
```

Comment:
I don't want to mention Chrome in this line, since it's talking just about the proposal itself, and we talk about Chrome status separately. Adding phase 4 makes sense. How about this:

```suggestion
The proposal is now in [finalized state (phase 4)](https://github.com/WebAssembly/simd/issues/480), both V8 and the toolchain have working implementations.
```

Reply:
Ok, sounds good,